[2018-05-27 14:28:43] local.ERROR: Symfony\Component\Debug\Exception\FatalThrowableError: Class 'Illuminate\Foundation\Console\EventGenerateCommand' not found in C:\wamp64\www\talentsaga\vendor\laravel\framework\src\Illuminate\Foundation\Providers\ArtisanServiceProvider.php:247
Stack trace:
#0 C:\wamp64\www\talentsaga\vendor\laravel\framework\src\Illuminate\Container\Container.php(731): Illuminate\Foundation\Providers\ArtisanServiceProvider->Illuminate\Foundation\Providers\{closure}(Object(Illuminate\Foundation\Application), Array)
#1 C:\wamp64\www\talentsaga\vendor\laravel\framework\src\Illuminate\Container\Container.php(629): Illuminate\Container\Container->build(Object(Closure), Array)
#2 C:\wamp64\www\talentsaga\vendor\laravel\framework\src\Illuminate\Foundation\Application.php(697): Illuminate\Container\Container->make('command.event.g...', Array)
#3 C:\wamp64\www\talentsaga\vendor\laravel\framework\src\Illuminate\Console\Application.php(115): Illuminate\Foundation\Application->make('command.event.g...')
#4 C:\wamp64\www\talentsaga\vendor\laravel\framework\src\Illuminate\Console\Application.php(129): Illuminate\Console\Application->resolve('command.event.g...')
#5 C:\wamp64\www\talentsaga\vendor\laravel\framework\src\Illuminate\Support\ServiceProvider.php(178): Illuminate\Console\Application->resolveCommands(Array)
#6 [internal function]: Illuminate\Support\ServiceProvider->Illuminate\Support\{closure}(Object(Illuminate\Console\Events\ArtisanStarting))
#7 C:\wamp64\www\talentsaga\vendor\laravel\framework\src\Illuminate\Events\Dispatcher.php(221): call_user_func_array(Object(Closure), Array)
#8 C:\wamp64\www\talentsaga\vendor\laravel\framework\src\Illuminate\Console\Application.php(46): Illuminate\Events\Dispatcher->fire('Illuminate\\Cons...')
#9 C:\wamp64\www\talentsaga\vendor\laravel\framework\src\Illuminate\Foundation\Console\Kernel.php(236): Illuminate\Console\Application->__construct(Object(Illuminate\Foundation\Application), Object(Illuminate\Events\Dispatcher), '5.2.45')
#10 C:\wamp64\www\talentsaga\vendor\laravel\framework\src\Illuminate\Foundation\Console\Kernel.php(107): Illuminate\Foundation\Console\Kernel->getArtisan()
#11 C:\wamp64\www\talentsaga\artisan(35): Illuminate\Foundation\Console\Kernel->handle(Object(Symfony\Component\Console\Input\ArgvInput), Object(Symfony\Component\Console\Output\ConsoleOutput))
#12 {main}  
[2018-05-27 14:38:09] local.ERROR: Symfony\Component\Debug\Exception\FatalThrowableError: Class 'Illuminate\Foundation\ConsoleSupportServiceProvider' not found in C:\wamp64\www\talentsaga\vendor\laravel\framework\src\Illuminate\Foundation\ProviderRepository.php:146
Stack trace:
#0 C:\wamp64\www\talentsaga\vendor\laravel\framework\src\Illuminate\Foundation\ProviderRepository.php(114): Illuminate\Foundation\ProviderRepository->createProvider('Illuminate\\Foun...')
#1 C:\wamp64\www\talentsaga\vendor\laravel\framework\src\Illuminate\Foundation\ProviderRepository.php(60): Illuminate\Foundation\ProviderRepository->compileManifest(Array)
#2 C:\wamp64\www\talentsaga\vendor\laravel\framework\src\Illuminate\Foundation\Application.php(530): Illuminate\Foundation\ProviderRepository->load(Array)
#3 C:\wamp64\www\talentsaga\vendor\laravel\framework\src\Illuminate\Foundation\Bootstrap\RegisterProviders.php(17): Illuminate\Foundation\Application->registerConfiguredProviders()
#4 C:\wamp64\www\talentsaga\vendor\laravel\framework\src\Illuminate\Foundation\Application.php(203): Illuminate\Foundation\Bootstrap\RegisterProviders->bootstrap(Object(Illuminate\Foundation\Application))
#5 C:\wamp64\www\talentsaga\vendor\laravel\framework\src\Illuminate\Foundation\Console\Kernel.php(219): Illuminate\Foundation\Application->bootstrapWith(Array)
#6 C:\wamp64\www\talentsaga\vendor\laravel\framework\src\Illuminate\Foundation\Console\Kernel.php(105): Illuminate\Foundation\Console\Kernel->bootstrap()
#7 C:\wamp64\www\talentsaga\artisan(35): Illuminate\Foundation\Console\Kernel->handle(Object(Symfony\Component\Console\Input\ArgvInput), Object(Symfony\Component\Console\Output\ConsoleOutput))
#8 {main}  
[2018-05-27 14:38:59] local.ERROR: Symfony\Component\Debug\Exception\FatalThrowableError: Class 'Illuminate\Foundation\Console\EventGenerateCommand' not found in C:\wamp64\www\talentsaga\vendor\laravel\framework\src\Illuminate\Foundation\Providers\ArtisanServiceProvider.php:247
Stack trace:
#0 C:\wamp64\www\talentsaga\vendor\laravel\framework\src\Illuminate\Container\Container.php(731): Illuminate\Foundation\Providers\ArtisanServiceProvider->Illuminate\Foundation\Providers\{closure}(Object(Illuminate\Foundation\Application), Array)
#1 C:\wamp64\www\talentsaga\vendor\laravel\framework\src\Illuminate\Container\Container.php(629): Illuminate\Container\Container->build(Object(Closure), Array)
#2 C:\wamp64\www\talentsaga\vendor\laravel\framework\src\Illuminate\Foundation\Application.php(697): Illuminate\Container\Container->make('command.event.g...', Array)
#3 C:\wamp64\www\talentsaga\vendor\laravel\framework\src\Illuminate\Console\Application.php(115): Illuminate\Foundation\Application->make('command.event.g...')
#4 C:\wamp64\www\talentsaga\vendor\laravel\framework\src\Illuminate\Console\Application.php(129): Illuminate\Console\Application->resolve('command.event.g...')
#5 C:\wamp64\www\talentsaga\vendor\laravel\framework\src\Illuminate\Support\ServiceProvider.php(178): Illuminate\Console\Application->resolveCommands(Array)
#6 [internal function]: Illuminate\Support\ServiceProvider->Illuminate\Support\{closure}(Object(Illuminate\Console\Events\ArtisanStarting))
#7 C:\wamp64\www\talentsaga\vendor\laravel\framework\src\Illuminate\Events\Dispatcher.php(221): call_user_func_array(Object(Closure), Array)
#8 C:\wamp64\www\talentsaga\vendor\laravel\framework\src\Illuminate\Console\Application.php(46): Illuminate\Events\Dispatcher->fire('Illuminate\\Cons...')
#9 C:\wamp64\www\talentsaga\vendor\laravel\framework\src\Illuminate\Foundation\Console\Kernel.php(236): Illuminate\Console\Application->__construct(Object(Illuminate\Foundation\Application), Object(Illuminate\Events\Dispatcher), '5.2.45')
#10 C:\wamp64\www\talentsaga\vendor\laravel\framework\src\Illuminate\Foundation\Console\Kernel.php(107): Illuminate\Foundation\Console\Kernel->getArtisan()
#11 C:\wamp64\www\talentsaga\artisan(35): Illuminate\Foundation\Console\Kernel->handle(Object(Symfony\Component\Console\Input\ArgvInput), Object(Symfony\Component\Console\Output\ConsoleOutput))
#12 {main}  
[2018-05-27 14:43:00] local.ERROR: Symfony\Component\Debug\Exception\FatalThrowableError: Class 'Illuminate\Foundation\ConsoleServiceProvider' not found in C:\wamp64\www\talentsaga\vendor\laravel\framework\src\Illuminate\Foundation\ProviderRepository.php:146
Stack trace:
#0 C:\wamp64\www\talentsaga\vendor\laravel\framework\src\Illuminate\Foundation\ProviderRepository.php(114): Illuminate\Foundation\ProviderRepository->createProvider('Illuminate\\Foun...')
#1 C:\wamp64\www\talentsaga\vendor\laravel\framework\src\Illuminate\Foundation\ProviderRepository.php(60): Illuminate\Foundation\ProviderRepository->compileManifest(Array)
#2 C:\wamp64\www\talentsaga\vendor\laravel\framework\src\Illuminate\Foundation\Application.php(530): Illuminate\Foundation\ProviderRepository->load(Array)
#3 C:\wamp64\www\talentsaga\vendor\laravel\framework\src\Illuminate\Foundation\Bootstrap\RegisterProviders.php(17): Illuminate\Foundation\Application->registerConfiguredProviders()
#4 C:\wamp64\www\talentsaga\vendor\laravel\framework\src\Illuminate\Foundation\Application.php(203): Illuminate\Foundation\Bootstrap\RegisterProviders->bootstrap(Object(Illuminate\Foundation\Application))
#5 C:\wamp64\www\talentsaga\vendor\laravel\framework\src\Illuminate\Foundation\Console\Kernel.php(219): Illuminate\Foundation\Application->bootstrapWith(Array)
#6 C:\wamp64\www\talentsaga\vendor\laravel\framework\src\Illuminate\Foundation\Console\Kernel.php(105): Illuminate\Foundation\Console\Kernel->bootstrap()
#7 C:\wamp64\www\talentsaga\artisan(35): Illuminate\Foundation\Console\Kernel->handle(Object(Symfony\Component\Console\Input\ArgvInput), Object(Symfony\Component\Console\Output\ConsoleOutput))
#8 {main}  
[2018-05-27 14:44:43] local.ERROR: Symfony\Component\Debug\Exception\FatalThrowableError: Class 'Illuminate\Foundation\ConsoleServiceProvider' not found in C:\wamp64\www\talentsaga\vendor\laravel\framework\src\Illuminate\Foundation\ProviderRepository.php:146
Stack trace:
#0 C:\wamp64\www\talentsaga\vendor\laravel\framework\src\Illuminate\Foundation\ProviderRepository.php(114): Illuminate\Foundation\ProviderRepository->createProvider('Illuminate\\Foun...')
#1 C:\wamp64\www\talentsaga\vendor\laravel\framework\src\Illuminate\Foundation\ProviderRepository.php(60): Illuminate\Foundation\ProviderRepository->compileManifest(Array)
#2 C:\wamp64\www\talentsaga\vendor\laravel\framework\src\Illuminate\Foundation\Application.php(530): Illuminate\Foundation\ProviderRepository->load(Array)
#3 C:\wamp64\www\talentsaga\vendor\laravel\framework\src\Illuminate\Foundation\Bootstrap\RegisterProviders.php(17): Illuminate\Foundation\Application->registerConfiguredProviders()
#4 C:\wamp64\www\talentsaga\vendor\laravel\framework\src\Illuminate\Foundation\Application.php(203): Illuminate\Foundation\Bootstrap\RegisterProviders->bootstrap(Object(Illuminate\Foundation\Application))
#5 C:\wamp64\www\talentsaga\vendor\laravel\framework\src\Illuminate\Foundation\Console\Kernel.php(219): Illuminate\Foundation\Application->bootstrapWith(Array)
#6 C:\wamp64\www\talentsaga\vendor\laravel\framework\src\Illuminate\Foundation\Console\Kernel.php(105): Illuminate\Foundation\Console\Kernel->bootstrap()
#7 C:\wamp64\www\talentsaga\artisan(35): Illuminate\Foundation\Console\Kernel->handle(Object(Symfony\Component\Console\Input\ArgvInput), Object(Symfony\Component\Console\Output\ConsoleOutput))
#8 {main}  
[2018-05-27 14:46:11] local.ERROR: Symfony\Component\Debug\Exception\FatalThrowableError: Class 'Illuminate\Foundation\Providers\ConsoleServiceProvider' not found in C:\wamp64\www\talentsaga\vendor\laravel\framework\src\Illuminate\Foundation\ProviderRepository.php:146
Stack trace:
#0 C:\wamp64\www\talentsaga\vendor\laravel\framework\src\Illuminate\Foundation\ProviderRepository.php(114): Illuminate\Foundation\ProviderRepository->createProvider('Illuminate\\Foun...')
#1 C:\wamp64\www\talentsaga\vendor\laravel\framework\src\Illuminate\Foundation\ProviderRepository.php(60): Illuminate\Foundation\ProviderRepository->compileManifest(Array)
#2 C:\wamp64\www\talentsaga\vendor\laravel\framework\src\Illuminate\Foundation\Application.php(530): Illuminate\Foundation\ProviderRepository->load(Array)
#3 C:\wamp64\www\talentsaga\vendor\laravel\framework\src\Illuminate\Foundation\Bootstrap\RegisterProviders.php(17): Illuminate\Foundation\Application->registerConfiguredProviders()
#4 C:\wamp64\www\talentsaga\vendor\laravel\framework\src\Illuminate\Foundation\Application.php(203): Illuminate\Foundation\Bootstrap\RegisterProviders->bootstrap(Object(Illuminate\Foundation\Application))
#5 C:\wamp64\www\talentsaga\vendor\laravel\framework\src\Illuminate\Foundation\Console\Kernel.php(219): Illuminate\Foundation\Application->bootstrapWith(Array)
#6 C:\wamp64\www\talentsaga\vendor\laravel\framework\src\Illuminate\Foundation\Console\Kernel.php(105): Illuminate\Foundation\Console\Kernel->bootstrap()
#7 C:\wamp64\www\talentsaga\artisan(35): Illuminate\Foundation\Console\Kernel->handle(Object(Symfony\Component\Console\Input\ArgvInput), Object(Symfony\Component\Console\Output\ConsoleOutput))
#8 {main}  
[2018-05-27 14:51:20] local.ERROR: Symfony\Component\Debug\Exception\FatalThrowableError: Class 'Illuminate\Foundation\Console\EventGenerateCommand' not found in C:\wamp64\www\talentsaga\vendor\laravel\framework\src\Illuminate\Foundation\Providers\ArtisanServiceProvider.php:247
Stack trace:
#0 C:\wamp64\www\talentsaga\vendor\laravel\framework\src\Illuminate\Container\Container.php(731): Illuminate\Foundation\Providers\ArtisanServiceProvider->Illuminate\Foundation\Providers\{closure}(Object(Illuminate\Foundation\Application), Array)
#1 C:\wamp64\www\talentsaga\vendor\laravel\framework\src\Illuminate\Container\Container.php(629): Illuminate\Container\Container->build(Object(Closure), Array)
#2 C:\wamp64\www\talentsaga\vendor\laravel\framework\src\Illuminate\Foundation\Application.php(697): Illuminate\Container\Container->make('command.event.g...', Array)
#3 C:\wamp64\www\talentsaga\vendor\laravel\framework\src\Illuminate\Console\Application.php(115): Illuminate\Foundation\Application->make('command.event.g...')
#4 C:\wamp64\www\talentsaga\vendor\laravel\framework\src\Illuminate\Console\Application.php(129): Illuminate\Console\Application->resolve('command.event.g...')
#5 C:\wamp64\www\talentsaga\vendor\laravel\framework\src\Illuminate\Support\ServiceProvider.php(178): Illuminate\Console\Application->resolveCommands(Array)
#6 [internal function]: Illuminate\Support\ServiceProvider->Illuminate\Support\{closure}(Object(Illuminate\Console\Events\ArtisanStarting))
#7 C:\wamp64\www\talentsaga\vendor\laravel\framework\src\Illuminate\Events\Dispatcher.php(221): call_user_func_array(Object(Closure), Array)
#8 C:\wamp64\www\talentsaga\vendor\laravel\framework\src\Illuminate\Console\Application.php(46): Illuminate\Events\Dispatcher->fire('Illuminate\\Cons...')
#9 C:\wamp64\www\talentsaga\vendor\laravel\framework\src\Illuminate\Foundation\Console\Kernel.php(236): Illuminate\Console\Application->__construct(Object(Illuminate\Foundation\Application), Object(Illuminate\Events\Dispatcher), '5.2.45')
#10 C:\wamp64\www\talentsaga\vendor\laravel\framework\src\Illuminate\Foundation\Console\Kernel.php(107): Illuminate\Foundation\Console\Kernel->getArtisan()
#11 C:\wamp64\www\talentsaga\artisan(35): Illuminate\Foundation\Console\Kernel->handle(Object(Symfony\Component\Console\Input\ArgvInput), Object(Symfony\Component\Console\Output\ConsoleOutput))
#12 {main}  
[2018-05-27 14:59:15] local.ERROR: Symfony\Component\Debug\Exception\FatalThrowableError: Class 'Illuminate\Foundation\Console\EventGenerateCommand' not found in C:\wamp64\www\talentsaga\vendor\laravel\framework\src\Illuminate\Foundation\Providers\ArtisanServiceProvider.php:247
Stack trace:
#0 C:\wamp64\www\talentsaga\vendor\laravel\framework\src\Illuminate\Container\Container.php(731): Illuminate\Foundation\Providers\ArtisanServiceProvider->Illuminate\Foundation\Providers\{closure}(Object(Illuminate\Foundation\Application), Array)
#1 C:\wamp64\www\talentsaga\vendor\laravel\framework\src\Illuminate\Container\Container.php(629): Illuminate\Container\Container->build(Object(Closure), Array)
#2 C:\wamp64\www\talentsaga\vendor\laravel\framework\src\Illuminate\Foundation\Application.php(697): Illuminate\Container\Container->make('command.event.g...', Array)
#3 C:\wamp64\www\talentsaga\vendor\laravel\framework\src\Illuminate\Console\Application.php(115): Illuminate\Foundation\Application->make('command.event.g...')
#4 C:\wamp64\www\talentsaga\vendor\laravel\framework\src\Illuminate\Console\Application.php(129): Illuminate\Console\Application->resolve('command.event.g...')
#5 C:\wamp64\www\talentsaga\vendor\laravel\framework\src\Illuminate\Support\ServiceProvider.php(178): Illuminate\Console\Application->resolveCommands(Array)
#6 [internal function]: Illuminate\Support\ServiceProvider->Illuminate\Support\{closure}(Object(Illuminate\Console\Events\ArtisanStarting))
#7 C:\wamp64\www\talentsaga\vendor\laravel\framework\src\Illuminate\Events\Dispatcher.php(221): call_user_func_array(Object(Closure), Array)
#8 C:\wamp64\www\talentsaga\vendor\laravel\framework\src\Illuminate\Console\Application.php(46): Illuminate\Events\Dispatcher->fire('Illuminate\\Cons...')
#9 C:\wamp64\www\talentsaga\vendor\laravel\framework\src\Illuminate\Foundation\Console\Kernel.php(236): Illuminate\Console\Application->__construct(Object(Illuminate\Foundation\Application), Object(Illuminate\Events\Dispatcher), '5.2.45')
#10 C:\wamp64\www\talentsaga\vendor\laravel\framework\src\Illuminate\Foundation\Console\Kernel.php(107): Illuminate\Foundation\Console\Kernel->getArtisan()
#11 C:\wamp64\www\talentsaga\artisan(35): Illuminate\Foundation\Console\Kernel->handle(Object(Symfony\Component\Console\Input\ArgvInput), Object(Symfony\Component\Console\Output\ConsoleOutput))
#12 {main}  
[2018-05-27 19:21:19] local.ERROR: Symfony\Component\Debug\Exception\FatalThrowableError: Class 'Illuminate\Foundation\Console\EventGenerateCommand' not found in C:\wamp64\www\talentsaga\vendor\laravel\framework\src\Illuminate\Foundation\Providers\ArtisanServiceProvider.php:247
Stack trace:
#0 C:\wamp64\www\talentsaga\vendor\laravel\framework\src\Illuminate\Container\Container.php(731): Illuminate\Foundation\Providers\ArtisanServiceProvider->Illuminate\Foundation\Providers\{closure}(Object(Illuminate\Foundation\Application), Array)
#1 C:\wamp64\www\talentsaga\vendor\laravel\framework\src\Illuminate\Container\Container.php(629): Illuminate\Container\Container->build(Object(Closure), Array)
#2 C:\wamp64\www\talentsaga\vendor\laravel\framework\src\Illuminate\Foundation\Application.php(697): Illuminate\Container\Container->make('command.event.g...', Array)
#3 C:\wamp64\www\talentsaga\vendor\laravel\framework\src\Illuminate\Console\Application.php(115): Illuminate\Foundation\Application->make('command.event.g...')
#4 C:\wamp64\www\talentsaga\vendor\laravel\framework\src\Illuminate\Console\Application.php(129): Illuminate\Console\Application->resolve('command.event.g...')
#5 C:\wamp64\www\talentsaga\vendor\laravel\framework\src\Illuminate\Support\ServiceProvider.php(178): Illuminate\Console\Application->resolveCommands(Array)
#6 [internal function]: Illuminate\Support\ServiceProvider->Illuminate\Support\{closure}(Object(Illuminate\Console\Events\ArtisanStarting))
#7 C:\wamp64\www\talentsaga\vendor\laravel\framework\src\Illuminate\Events\Dispatcher.php(221): call_user_func_array(Object(Closure), Array)
#8 C:\wamp64\www\talentsaga\vendor\laravel\framework\src\Illuminate\Console\Application.php(46): Illuminate\Events\Dispatcher->fire('Illuminate\\Cons...')
#9 C:\wamp64\www\talentsaga\vendor\laravel\framework\src\Illuminate\Foundation\Console\Kernel.php(236): Illuminate\Console\Application->__construct(Object(Illuminate\Foundation\Application), Object(Illuminate\Events\Dispatcher), '5.2.45')
#10 C:\wamp64\www\talentsaga\vendor\laravel\framework\src\Illuminate\Foundation\Console\Kernel.php(107): Illuminate\Foundation\Console\Kernel->getArtisan()
#11 C:\wamp64\www\talentsaga\artisan(35): Illuminate\Foundation\Console\Kernel->handle(Object(Symfony\Component\Console\Input\ArgvInput), Object(Symfony\Component\Console\Output\ConsoleOutput))
#12 {main}  
[2018-05-28 21:33:56] production.ERROR: exception 'Symfony\Component\Debug\Exception\FatalErrorException' with message 'Method App\Models\City::__toString() must not throw an exception' in C:\wamp64\www\talentsaga\vendor\laravel\framework\src\Illuminate\Support\helpers.php:0
Stack trace:
#0 C:\wamp64\www\talentsaga\vendor\laravel\framework\src\Illuminate\Foundation\Bootstrap\HandleExceptions.php(133): Symfony\Component\Debug\Exception\FatalErrorException->__construct()
#1 C:\wamp64\www\talentsaga\vendor\laravel\framework\src\Illuminate\Foundation\Bootstrap\HandleExceptions.php(118): Illuminate\Foundation\Bootstrap\HandleExceptions->fatalExceptionFromError()
#2 C:\wamp64\www\talentsaga\vendor\laravel\framework\src\Illuminate\Foundation\Bootstrap\HandleExceptions.php(0): Illuminate\Foundation\Bootstrap\HandleExceptions->handleShutdown()
#3 C:\wamp64\www\talentsaga\vendor\laravel\framework\src\Illuminate\Support\helpers.php(531): htmlentities()
#4 C:\wamp64\www\talentsaga\storage\framework\views\8d8165299453625bac9b575ae33b1982c6ded2c4.php(195): e()
#5 C:\wamp64\www\talentsaga\vendor\laravel\framework\src\Illuminate\View\Engines\PhpEngine.php(42): ()
#6 C:\wamp64\www\talentsaga\vendor\laravel\framework\src\Illuminate\View\Engines\CompilerEngine.php(59): Illuminate\View\Engines\PhpEngine->evaluatePath()
#7 C:\wamp64\www\talentsaga\vendor\laravel\framework\src\Illuminate\View\View.php(149): Illuminate\View\Engines\CompilerEngine->get()
#8 C:\wamp64\www\talentsaga\vendor\laravel\framework\src\Illuminate\View\View.php(120): Illuminate\View\View->getContents()
#9 C:\wamp64\www\talentsaga\vendor\laravel\framework\src\Illuminate\View\View.php(85): Illuminate\View\View->renderContents()
#10 C:\wamp64\www\talentsaga\vendor\laravel\framework\src\Illuminate\Http\Response.php(53): Illuminate\View\View->render()
#11 C:\wamp64\www\talentsaga\vendor\symfony\http-foundation\Response.php(201): Illuminate\Http\Response->setContent()
#12 C:\wamp64\www\talentsaga\vendor\laravel\framework\src\Illuminate\Routing\ResponseFactory.php(57): Symfony\Component\HttpFoundation\Response->__construct()
#13 C:\wamp64\www\talentsaga\vendor\laravel\framework\src\Illuminate\Routing\ResponseFactory.php(71): Illuminate\Routing\ResponseFactory->make()
#14 C:\wamp64\www\talentsaga\vendor\laravel\framework\src\Illuminate\Foundation\Exceptions\Handler.php(156): Illuminate\Routing\ResponseFactory->view()
#15 C:\wamp64\www\talentsaga\vendor\laravel\framework\src\Illuminate\Foundation\Exceptions\Handler.php(113): Illuminate\Foundation\Exceptions\Handler->renderHttpException()
#16 C:\wamp64\www\talentsaga\app\Exceptions\Handler.php(52): Illuminate\Foundation\Exceptions\Handler->render()
#17 C:\wamp64\www\talentsaga\vendor\laravel\framework\src\Illuminate\Routing\Pipeline.php(80): App\Exceptions\Handler->render()
#18 C:\wamp64\www\talentsaga\vendor\laravel\framework\src\Illuminate\Routing\Pipeline.php(54): Illuminate\Routing\Pipeline->handleException()
#19 C:\wamp64\www\talentsaga\vendor\laravel\framework\src\Illuminate\Foundation\Http\Middleware\CheckForMaintenanceMode.php(44): Illuminate\Routing\Pipeline->Illuminate\Routing\{closure}()
#20 C:\wamp64\www\talentsaga\vendor\laravel\framework\src\Illuminate\Pipeline\Pipeline.php(136): Illuminate\Foundation\Http\Middleware\CheckForMaintenanceMode->handle()
#21 C:\wamp64\www\talentsaga\vendor\laravel\framework\src\Illuminate\Pipeline\Pipeline.php(136): call_user_func_array:{C:\wamp64\www\talentsaga\vendor\laravel\framework\src\Illuminate\Pipeline\Pipeline.php:136}()
#22 C:\wamp64\www\talentsaga\vendor\laravel\framework\src\Illuminate\Routing\Pipeline.php(32): Illuminate\Pipeline\Pipeline->Illuminate\Pipeline\{closure}()
#23 C:\wamp64\www\talentsaga\vendor\laravel\framework\src\Illuminate\Routing\Pipeline.php(32): call_user_func:{C:\wamp64\www\talentsaga\vendor\laravel\framework\src\Illuminate\Routing\Pipeline.php:32}()
#24 C:\wamp64\www\talentsaga\vendor\laravel\framework\src\Illuminate\Pipeline\Pipeline.php(103): Illuminate\Routing\Pipeline->Illuminate\Routing\{closure}()
#25 C:\wamp64\www\talentsaga\vendor\laravel\framework\src\Illuminate\Pipeline\Pipeline.php(103): call_user_func:{C:\wamp64\www\talentsaga\vendor\laravel\framework\src\Illuminate\Pipeline\Pipeline.php:103}()
#26 C:\wamp64\www\talentsaga\vendor\laravel\framework\src\Illuminate\Foundation\Http\Kernel.php(132): Illuminate\Pipeline\Pipeline->then()
#27 C:\wamp64\www\talentsaga\vendor\laravel\framework\src\Illuminate\Foundation\Http\Kernel.php(99): Illuminate\Foundation\Http\Kernel->sendRequestThroughRouter()
#28 C:\wamp64\www\talentsaga\public\index.php(54): Illuminate\Foundation\Http\Kernel->handle()
#29 C:\wamp64\www\talentsaga\public\index.php(0): {main}()
#30 {main}  
[2018-05-28 21:33:56] production.ERROR: exception 'Symfony\Component\Debug\Exception\FatalErrorException' with message 'Method App\Models\City::__toString() must not throw an exception' in C:\wamp64\www\talentsaga\vendor\laravel\framework\src\Illuminate\Support\helpers.php:0
Stack trace:
#0 C:\wamp64\www\talentsaga\vendor\laravel\framework\src\Illuminate\Foundation\Bootstrap\HandleExceptions.php(133): Symfony\Component\Debug\Exception\FatalErrorException->__construct()
#1 C:\wamp64\www\talentsaga\vendor\laravel\framework\src\Illuminate\Foundation\Bootstrap\HandleExceptions.php(118): Illuminate\Foundation\Bootstrap\HandleExceptions->fatalExceptionFromError()
#2 C:\wamp64\www\talentsaga\vendor\laravel\framework\src\Illuminate\Foundation\Bootstrap\HandleExceptions.php(0): Illuminate\Foundation\Bootstrap\HandleExceptions->handleShutdown()
#3 C:\wamp64\www\talentsaga\vendor\laravel\framework\src\Illuminate\Support\helpers.php(531): htmlentities()
#4 C:\wamp64\www\talentsaga\storage\framework\views\8d8165299453625bac9b575ae33b1982c6ded2c4.php(195): e()
#5 C:\wamp64\www\talentsaga\vendor\laravel\framework\src\Illuminate\View\Engines\PhpEngine.php(42): ()
#6 C:\wamp64\www\talentsaga\vendor\laravel\framework\src\Illuminate\View\Engines\CompilerEngine.php(59): Illuminate\View\Engines\PhpEngine->evaluatePath()
#7 C:\wamp64\www\talentsaga\vendor\laravel\framework\src\Illuminate\View\View.php(149): Illuminate\View\Engines\CompilerEngine->get()
#8 C:\wamp64\www\talentsaga\vendor\laravel\framework\src\Illuminate\View\View.php(120): Illuminate\View\View->getContents()
#9 C:\wamp64\www\talentsaga\vendor\laravel\framework\src\Illuminate\View\View.php(85): Illuminate\View\View->renderContents()
#10 C:\wamp64\www\talentsaga\vendor\laravel\framework\src\Illuminate\Http\Response.php(53): Illuminate\View\View->render()
#11 C:\wamp64\www\talentsaga\vendor\symfony\http-foundation\Response.php(201): Illuminate\Http\Response->setContent()
#12 C:\wamp64\www\talentsaga\vendor\laravel\framework\src\Illuminate\Routing\ResponseFactory.php(57): Symfony\Component\HttpFoundation\Response->__construct()
#13 C:\wamp64\www\talentsaga\vendor\laravel\framework\src\Illuminate\Routing\ResponseFactory.php(71): Illuminate\Routing\ResponseFactory->make()
#14 C:\wamp64\www\talentsaga\vendor\laravel\framework\src\Illuminate\Foundation\Exceptions\Handler.php(156): Illuminate\Routing\ResponseFactory->view()
#15 C:\wamp64\www\talentsaga\vendor\laravel\framework\src\Illuminate\Foundation\Exceptions\Handler.php(113): Illuminate\Foundation\Exceptions\Handler->renderHttpException()
#16 C:\wamp64\www\talentsaga\app\Exceptions\Handler.php(52): Illuminate\Foundation\Exceptions\Handler->render()
#17 C:\wamp64\www\talentsaga\vendor\laravel\framework\src\Illuminate\Routing\Pipeline.php(80): App\Exceptions\Handler->render()
#18 C:\wamp64\www\talentsaga\vendor\laravel\framework\src\Illuminate\Routing\Pipeline.php(54): Illuminate\Routing\Pipeline->handleException()
#19 C:\wamp64\www\talentsaga\vendor\laravel\framework\src\Illuminate\Foundation\Http\Middleware\CheckForMaintenanceMode.php(44): Illuminate\Routing\Pipeline->Illuminate\Routing\{closure}()
#20 C:\wamp64\www\talentsaga\vendor\laravel\framework\src\Illuminate\Pipeline\Pipeline.php(136): Illuminate\Foundation\Http\Middleware\CheckForMaintenanceMode->handle()
#21 C:\wamp64\www\talentsaga\vendor\laravel\framework\src\Illuminate\Pipeline\Pipeline.php(136): call_user_func_array:{C:\wamp64\www\talentsaga\vendor\laravel\framework\src\Illuminate\Pipeline\Pipeline.php:136}()
#22 C:\wamp64\www\talentsaga\vendor\laravel\framework\src\Illuminate\Routing\Pipeline.php(32): Illuminate\Pipeline\Pipeline->Illuminate\Pipeline\{closure}()
#23 C:\wamp64\www\talentsaga\vendor\laravel\framework\src\Illuminate\Routing\Pipeline.php(32): call_user_func:{C:\wamp64\www\talentsaga\vendor\laravel\framework\src\Illuminate\Routing\Pipeline.php:32}()
#24 C:\wamp64\www\talentsaga\vendor\laravel\framework\src\Illuminate\Pipeline\Pipeline.php(103): Illuminate\Routing\Pipeline->Illuminate\Routing\{closure}()
#25 C:\wamp64\www\talentsaga\vendor\laravel\framework\src\Illuminate\Pipeline\Pipeline.php(103): call_user_func:{C:\wamp64\www\talentsaga\vendor\laravel\framework\src\Illuminate\Pipeline\Pipeline.php:103}()
#26 C:\wamp64\www\talentsaga\vendor\laravel\framework\src\Illuminate\Foundation\Http\Kernel.php(132): Illuminate\Pipeline\Pipeline->then()
#27 C:\wamp64\www\talentsaga\vendor\laravel\framework\src\Illuminate\Foundation\Http\Kernel.php(99): Illuminate\Foundation\Http\Kernel->sendRequestThroughRouter()
#28 C:\wamp64\www\talentsaga\public\index.php(54): Illuminate\Foundation\Http\Kernel->handle()
#29 C:\wamp64\www\talentsaga\public\index.php(0): {main}()
#30 {main}  
[2018-05-28 21:33:56] production.ERROR: exception 'Symfony\Component\Debug\Exception\FatalErrorException' with message 'Method App\Models\City::__toString() must not throw an exception' in C:\wamp64\www\talentsaga\vendor\laravel\framework\src\Illuminate\Support\helpers.php:0
Stack trace:
#0 C:\wamp64\www\talentsaga\vendor\laravel\framework\src\Illuminate\Foundation\Bootstrap\HandleExceptions.php(133): Symfony\Component\Debug\Exception\FatalErrorException->__construct()
#1 C:\wamp64\www\talentsaga\vendor\laravel\framework\src\Illuminate\Foundation\Bootstrap\HandleExceptions.php(118): Illuminate\Foundation\Bootstrap\HandleExceptions->fatalExceptionFromError()
#2 C:\wamp64\www\talentsaga\vendor\laravel\framework\src\Illuminate\Foundation\Bootstrap\HandleExceptions.php(0): Illuminate\Foundation\Bootstrap\HandleExceptions->handleShutdown()
#3 C:\wamp64\www\talentsaga\vendor\laravel\framework\src\Illuminate\Support\helpers.php(531): htmlentities()
#4 C:\wamp64\www\talentsaga\storage\framework\views\8d8165299453625bac9b575ae33b1982c6ded2c4.php(195): e()
#5 C:\wamp64\www\talentsaga\vendor\laravel\framework\src\Illuminate\View\Engines\PhpEngine.php(42): ()
#6 C:\wamp64\www\talentsaga\vendor\laravel\framework\src\Illuminate\View\Engines\CompilerEngine.php(59): Illuminate\View\Engines\PhpEngine->evaluatePath()
#7 C:\wamp64\www\talentsaga\vendor\laravel\framework\src\Illuminate\View\View.php(149): Illuminate\View\Engines\CompilerEngine->get()
#8 C:\wamp64\www\talentsaga\vendor\laravel\framework\src\Illuminate\View\View.php(120): Illuminate\View\View->getContents()
#9 C:\wamp64\www\talentsaga\vendor\laravel\framework\src\Illuminate\View\View.php(85): Illuminate\View\View->renderContents()
#10 C:\wamp64\www\talentsaga\vendor\laravel\framework\src\Illuminate\Http\Response.php(53): Illuminate\View\View->render()
#11 C:\wamp64\www\talentsaga\vendor\symfony\http-foundation\Response.php(201): Illuminate\Http\Response->setContent()
#12 C:\wamp64\www\talentsaga\vendor\laravel\framework\src\Illuminate\Routing\ResponseFactory.php(57): Symfony\Component\HttpFoundation\Response->__construct()
#13 C:\wamp64\www\talentsaga\vendor\laravel\framework\src\Illuminate\Routing\ResponseFactory.php(71): Illuminate\Routing\ResponseFactory->make()
#14 C:\wamp64\www\talentsaga\vendor\laravel\framework\src\Illuminate\Foundation\Exceptions\Handler.php(156): Illuminate\Routing\ResponseFactory->view()
#15 C:\wamp64\www\talentsaga\vendor\laravel\framework\src\Illuminate\Foundation\Exceptions\Handler.php(113): Illuminate\Foundation\Exceptions\Handler->renderHttpException()
#16 C:\wamp64\www\talentsaga\app\Exceptions\Handler.php(52): Illuminate\Foundation\Exceptions\Handler->render()
#17 C:\wamp64\www\talentsaga\vendor\laravel\framework\src\Illuminate\Routing\Pipeline.php(80): App\Exceptions\Handler->render()
#18 C:\wamp64\www\talentsaga\vendor\laravel\framework\src\Illuminate\Routing\Pipeline.php(54): Illuminate\Routing\Pipeline->handleException()
#19 C:\wamp64\www\talentsaga\vendor\laravel\framework\src\Illuminate\Foundation\Http\Middleware\CheckForMaintenanceMode.php(44): Illuminate\Routing\Pipeline->Illuminate\Routing\{closure}()
#20 C:\wamp64\www\talentsaga\vendor\laravel\framework\src\Illuminate\Pipeline\Pipeline.php(136): Illuminate\Foundation\Http\Middleware\CheckForMaintenanceMode->handle()
#21 C:\wamp64\www\talentsaga\vendor\laravel\framework\src\Illuminate\Pipeline\Pipeline.php(136): call_user_func_array:{C:\wamp64\www\talentsaga\vendor\laravel\framework\src\Illuminate\Pipeline\Pipeline.php:136}()
#22 C:\wamp64\www\talentsaga\vendor\laravel\framework\src\Illuminate\Routing\Pipeline.php(32): Illuminate\Pipeline\Pipeline->Illuminate\Pipeline\{closure}()
#23 C:\wamp64\www\talentsaga\vendor\laravel\framework\src\Illuminate\Routing\Pipeline.php(32): call_user_func:{C:\wamp64\www\talentsaga\vendor\laravel\framework\src\Illuminate\Routing\Pipeline.php:32}()
#24 C:\wamp64\www\talentsaga\vendor\laravel\framework\src\Illuminate\Pipeline\Pipeline.php(103): Illuminate\Routing\Pipeline->Illuminate\Routing\{closure}()
#25 C:\wamp64\www\talentsaga\vendor\laravel\framework\src\Illuminate\Pipeline\Pipeline.php(103): call_user_func:{C:\wamp64\www\talentsaga\vendor\laravel\framework\src\Illuminate\Pipeline\Pipeline.php:103}()
#26 C:\wamp64\www\talentsaga\vendor\laravel\framework\src\Illuminate\Foundation\Http\Kernel.php(132): Illuminate\Pipeline\Pipeline->then()
#27 C:\wamp64\www\talentsaga\vendor\laravel\framework\src\Illuminate\Foundation\Http\Kernel.php(99): Illuminate\Foundation\Http\Kernel->sendRequestThroughRouter()
#28 C:\wamp64\www\talentsaga\public\index.php(54): Illuminate\Foundation\Http\Kernel->handle()
#29 C:\wamp64\www\talentsaga\public\index.php(0): {main}()
#30 {main}  
[2018-05-31 17:04:20] production.ERROR: exception 'Symfony\Component\Debug\Exception\FatalErrorException' with message 'Method App\Models\City::__toString() must not throw an exception' in C:\wamp64\www\talentsaga\vendor\laravel\framework\src\Illuminate\Support\helpers.php:0
Stack trace:
#0 C:\wamp64\www\talentsaga\vendor\laravel\framework\src\Illuminate\Foundation\Bootstrap\HandleExceptions.php(133): Symfony\Component\Debug\Exception\FatalErrorException->__construct()
#1 C:\wamp64\www\talentsaga\vendor\laravel\framework\src\Illuminate\Foundation\Bootstrap\HandleExceptions.php(118): Illuminate\Foundation\Bootstrap\HandleExceptions->fatalExceptionFromError()
#2 C:\wamp64\www\talentsaga\vendor\laravel\framework\src\Illuminate\Foundation\Bootstrap\HandleExceptions.php(0): Illuminate\Foundation\Bootstrap\HandleExceptions->handleShutdown()
#3 C:\wamp64\www\talentsaga\vendor\laravel\framework\src\Illuminate\Support\helpers.php(531): htmlentities()
#4 C:\wamp64\www\talentsaga\storage\framework\views\8d8165299453625bac9b575ae33b1982c6ded2c4.php(195): e()
#5 C:\wamp64\www\talentsaga\vendor\laravel\framework\src\Illuminate\View\Engines\PhpEngine.php(42): ()
#6 C:\wamp64\www\talentsaga\vendor\laravel\framework\src\Illuminate\View\Engines\CompilerEngine.php(59): Illuminate\View\Engines\PhpEngine->evaluatePath()
#7 C:\wamp64\www\talentsaga\vendor\laravel\framework\src\Illuminate\View\View.php(149): Illuminate\View\Engines\CompilerEngine->get()
#8 C:\wamp64\www\talentsaga\vendor\laravel\framework\src\Illuminate\View\View.php(120): Illuminate\View\View->getContents()
#9 C:\wamp64\www\talentsaga\vendor\laravel\framework\src\Illuminate\View\View.php(85): Illuminate\View\View->renderContents()
#10 C:\wamp64\www\talentsaga\vendor\laravel\framework\src\Illuminate\Http\Response.php(53): Illuminate\View\View->render()
#11 C:\wamp64\www\talentsaga\vendor\symfony\http-foundation\Response.php(201): Illuminate\Http\Response->setContent()
#12 C:\wamp64\www\talentsaga\vendor\laravel\framework\src\Illuminate\Routing\ResponseFactory.php(57): Symfony\Component\HttpFoundation\Response->__construct()
#13 C:\wamp64\www\talentsaga\vendor\laravel\framework\src\Illuminate\Routing\ResponseFactory.php(71): Illuminate\Routing\ResponseFactory->make()
#14 C:\wamp64\www\talentsaga\vendor\laravel\framework\src\Illuminate\Foundation\Exceptions\Handler.php(156): Illuminate\Routing\ResponseFactory->view()
#15 C:\wamp64\www\talentsaga\vendor\laravel\framework\src\Illuminate\Foundation\Exceptions\Handler.php(113): Illuminate\Foundation\Exceptions\Handler->renderHttpException()
#16 C:\wamp64\www\talentsaga\app\Exceptions\Handler.php(52): Illuminate\Foundation\Exceptions\Handler->render()
#17 C:\wamp64\www\talentsaga\vendor\laravel\framework\src\Illuminate\Routing\Pipeline.php(80): App\Exceptions\Handler->render()
#18 C:\wamp64\www\talentsaga\vendor\laravel\framework\src\Illuminate\Routing\Pipeline.php(54): Illuminate\Routing\Pipeline->handleException()
#19 C:\wamp64\www\talentsaga\vendor\laravel\framework\src\Illuminate\Foundation\Http\Middleware\CheckForMaintenanceMode.php(44): Illuminate\Routing\Pipeline->Illuminate\Routing\{closure}()
#20 C:\wamp64\www\talentsaga\vendor\laravel\framework\src\Illuminate\Pipeline\Pipeline.php(136): Illuminate\Foundation\Http\Middleware\CheckForMaintenanceMode->handle()
#21 C:\wamp64\www\talentsaga\vendor\laravel\framework\src\Illuminate\Pipeline\Pipeline.php(136): call_user_func_array:{C:\wamp64\www\talentsaga\vendor\laravel\framework\src\Illuminate\Pipeline\Pipeline.php:136}()
#22 C:\wamp64\www\talentsaga\vendor\laravel\framework\src\Illuminate\Routing\Pipeline.php(32): Illuminate\Pipeline\Pipeline->Illuminate\Pipeline\{closure}()
#23 C:\wamp64\www\talentsaga\vendor\laravel\framework\src\Illuminate\Routing\Pipeline.php(32): call_user_func:{C:\wamp64\www\talentsaga\vendor\laravel\framework\src\Illuminate\Routing\Pipeline.php:32}()
#24 C:\wamp64\www\talentsaga\vendor\laravel\framework\src\Illuminate\Pipeline\Pipeline.php(103): Illuminate\Routing\Pipeline->Illuminate\Routing\{closure}()
#25 C:\wamp64\www\talentsaga\vendor\laravel\framework\src\Illuminate\Pipeline\Pipeline.php(103): call_user_func:{C:\wamp64\www\talentsaga\vendor\laravel\framework\src\Illuminate\Pipeline\Pipeline.php:103}()
#26 C:\wamp64\www\talentsaga\vendor\laravel\framework\src\Illuminate\Foundation\Http\Kernel.php(132): Illuminate\Pipeline\Pipeline->then()
#27 C:\wamp64\www\talentsaga\vendor\laravel\framework\src\Illuminate\Foundation\Http\Kernel.php(99): Illuminate\Foundation\Http\Kernel->sendRequestThroughRouter()
#28 C:\wamp64\www\talentsaga\public\index.php(54): Illuminate\Foundation\Http\Kernel->handle()
#29 C:\wamp64\www\talentsaga\server.php(21): ()
#30 C:\wamp64\www\talentsaga\server.php(0): {main}()
#31 {main}  
[2018-05-31 17:04:20] production.ERROR: exception 'Symfony\Component\Debug\Exception\FatalErrorException' with message 'Method App\Models\City::__toString() must not throw an exception' in C:\wamp64\www\talentsaga\vendor\laravel\framework\src\Illuminate\Support\helpers.php:0
Stack trace:
#0 C:\wamp64\www\talentsaga\vendor\laravel\framework\src\Illuminate\Foundation\Bootstrap\HandleExceptions.php(133): Symfony\Component\Debug\Exception\FatalErrorException->__construct()
#1 C:\wamp64\www\talentsaga\vendor\laravel\framework\src\Illuminate\Foundation\Bootstrap\HandleExceptions.php(118): Illuminate\Foundation\Bootstrap\HandleExceptions->fatalExceptionFromError()
#2 C:\wamp64\www\talentsaga\vendor\laravel\framework\src\Illuminate\Foundation\Bootstrap\HandleExceptions.php(0): Illuminate\Foundation\Bootstrap\HandleExceptions->handleShutdown()
#3 C:\wamp64\www\talentsaga\vendor\laravel\framework\src\Illuminate\Support\helpers.php(531): htmlentities()
#4 C:\wamp64\www\talentsaga\storage\framework\views\8d8165299453625bac9b575ae33b1982c6ded2c4.php(195): e()
#5 C:\wamp64\www\talentsaga\vendor\laravel\framework\src\Illuminate\View\Engines\PhpEngine.php(42): ()
#6 C:\wamp64\www\talentsaga\vendor\laravel\framework\src\Illuminate\View\Engines\CompilerEngine.php(59): Illuminate\View\Engines\PhpEngine->evaluatePath()
#7 C:\wamp64\www\talentsaga\vendor\laravel\framework\src\Illuminate\View\View.php(149): Illuminate\View\Engines\CompilerEngine->get()
#8 C:\wamp64\www\talentsaga\vendor\laravel\framework\src\Illuminate\View\View.php(120): Illuminate\View\View->getContents()
#9 C:\wamp64\www\talentsaga\vendor\laravel\framework\src\Illuminate\View\View.php(85): Illuminate\View\View->renderContents()
#10 C:\wamp64\www\talentsaga\vendor\laravel\framework\src\Illuminate\Http\Response.php(53): Illuminate\View\View->render()
#11 C:\wamp64\www\talentsaga\vendor\symfony\http-foundation\Response.php(201): Illuminate\Http\Response->setContent()
#12 C:\wamp64\www\talentsaga\vendor\laravel\framework\src\Illuminate\Routing\ResponseFactory.php(57): Symfony\Component\HttpFoundation\Response->__construct()
#13 C:\wamp64\www\talentsaga\vendor\laravel\framework\src\Illuminate\Routing\ResponseFactory.php(71): Illuminate\Routing\ResponseFactory->make()
#14 C:\wamp64\www\talentsaga\vendor\laravel\framework\src\Illuminate\Foundation\Exceptions\Handler.php(156): Illuminate\Routing\ResponseFactory->view()
#15 C:\wamp64\www\talentsaga\vendor\laravel\framework\src\Illuminate\Foundation\Exceptions\Handler.php(113): Illuminate\Foundation\Exceptions\Handler->renderHttpException()
#16 C:\wamp64\www\talentsaga\app\Exceptions\Handler.php(52): Illuminate\Foundation\Exceptions\Handler->render()
#17 C:\wamp64\www\talentsaga\vendor\laravel\framework\src\Illuminate\Routing\Pipeline.php(80): App\Exceptions\Handler->render()
#18 C:\wamp64\www\talentsaga\vendor\laravel\framework\src\Illuminate\Routing\Pipeline.php(54): Illuminate\Routing\Pipeline->handleException()
#19 C:\wamp64\www\talentsaga\vendor\laravel\framework\src\Illuminate\Foundation\Http\Middleware\CheckForMaintenanceMode.php(44): Illuminate\Routing\Pipeline->Illuminate\Routing\{closure}()
#20 C:\wamp64\www\talentsaga\vendor\laravel\framework\src\Illuminate\Pipeline\Pipeline.php(136): Illuminate\Foundation\Http\Middleware\CheckForMaintenanceMode->handle()
#21 C:\wamp64\www\talentsaga\vendor\laravel\framework\src\Illuminate\Pipeline\Pipeline.php(136): call_user_func_array:{C:\wamp64\www\talentsaga\vendor\laravel\framework\src\Illuminate\Pipeline\Pipeline.php:136}()
#22 C:\wamp64\www\talentsaga\vendor\laravel\framework\src\Illuminate\Routing\Pipeline.php(32): Illuminate\Pipeline\Pipeline->Illuminate\Pipeline\{closure}()
#23 C:\wamp64\www\talentsaga\vendor\laravel\framework\src\Illuminate\Routing\Pipeline.php(32): call_user_func:{C:\wamp64\www\talentsaga\vendor\laravel\framework\src\Illuminate\Routing\Pipeline.php:32}()
#24 C:\wamp64\www\talentsaga\vendor\laravel\framework\src\Illuminate\Pipeline\Pipeline.php(103): Illuminate\Routing\Pipeline->Illuminate\Routing\{closure}()
#25 C:\wamp64\www\talentsaga\vendor\laravel\framework\src\Illuminate\Pipeline\Pipeline.php(103): call_user_func:{C:\wamp64\www\talentsaga\vendor\laravel\framework\src\Illuminate\Pipeline\Pipeline.php:103}()
#26 C:\wamp64\www\talentsaga\vendor\laravel\framework\src\Illuminate\Foundation\Http\Kernel.php(132): Illuminate\Pipeline\Pipeline->then()
#27 C:\wamp64\www\talentsaga\vendor\laravel\framework\src\Illuminate\Foundation\Http\Kernel.php(99): Illuminate\Foundation\Http\Kernel->sendRequestThroughRouter()
#28 C:\wamp64\www\talentsaga\public\index.php(54): Illuminate\Foundation\Http\Kernel->handle()
#29 C:\wamp64\www\talentsaga\server.php(21): ()
#30 C:\wamp64\www\talentsaga\server.php(0): {main}()
#31 {main}  
[2018-06-01 09:48:46] local.ERROR: exception 'PDOException' with message 'SQLSTATE[HY000] [1049] Unknown database 'forge'' in D:\talentsaga\vendor\laravel\framework\src\Illuminate\Database\Connectors\Connector.php:55
Stack trace:
#0 D:\talentsaga\vendor\laravel\framework\src\Illuminate\Database\Connectors\Connector.php(55): PDO->__construct('mysql:host=loca...', 'root', '', Array)
#1 D:\talentsaga\vendor\laravel\framework\src\Illuminate\Database\Connectors\MySqlConnector.php(24): Illuminate\Database\Connectors\Connector->createConnection('mysql:host=loca...', Array, Array)
#2 D:\talentsaga\vendor\laravel\framework\src\Illuminate\Database\Connectors\ConnectionFactory.php(61): Illuminate\Database\Connectors\MySqlConnector->connect(Array)
#3 [internal function]: Illuminate\Database\Connectors\ConnectionFactory->Illuminate\Database\Connectors\{closure}()
#4 D:\talentsaga\vendor\laravel\framework\src\Illuminate\Database\Connection.php(924): call_user_func(Object(Closure))
#5 D:\talentsaga\vendor\laravel\framework\src\Illuminate\Database\Connection.php(792): Illuminate\Database\Connection->getPdo()
#6 D:\talentsaga\vendor\laravel\framework\src\Illuminate\Database\Connection.php(677): Illuminate\Database\Connection->reconnectIfMissingConnection()
#7 D:\talentsaga\vendor\laravel\framework\src\Illuminate\Database\Connection.php(349): Illuminate\Database\Connection->run('select * from `...', Array, Object(Closure))
#8 D:\talentsaga\vendor\laravel\framework\src\Illuminate\Database\Query\Builder.php(1610): Illuminate\Database\Connection->select('select * from `...', Array, true)
#9 D:\talentsaga\vendor\laravel\framework\src\Illuminate\Database\Query\Builder.php(1596): Illuminate\Database\Query\Builder->runSelect()
#10 D:\talentsaga\vendor\laravel\framework\src\Illuminate\Database\Eloquent\Builder.php(625): Illuminate\Database\Query\Builder->get(Array)
#11 D:\talentsaga\vendor\laravel\framework\src\Illuminate\Database\Eloquent\Builder.php(316): Illuminate\Database\Eloquent\Builder->getModels(Array)
#12 D:\talentsaga\app\Http\Controllers\Frontend\HomeController.php(65): Illuminate\Database\Eloquent\Builder->get()
#13 [internal function]: App\Http\Controllers\Frontend\HomeController->getIndex()
#14 D:\talentsaga\vendor\laravel\framework\src\Illuminate\Routing\Controller.php(80): call_user_func_array(Array, Array)
#15 D:\talentsaga\vendor\laravel\framework\src\Illuminate\Routing\ControllerDispatcher.php(146): Illuminate\Routing\Controller->callAction('getIndex', Array)
#16 D:\talentsaga\vendor\laravel\framework\src\Illuminate\Routing\ControllerDispatcher.php(94): Illuminate\Routing\ControllerDispatcher->call(Object(App\Http\Controllers\Frontend\HomeController), Object(Illuminate\Routing\Route), 'getIndex')
#17 [internal function]: Illuminate\Routing\ControllerDispatcher->Illuminate\Routing\{closure}(Object(Illuminate\Http\Request))
#18 D:\talentsaga\vendor\laravel\framework\src\Illuminate\Routing\Pipeline.php(52): call_user_func(Object(Closure), Object(Illuminate\Http\Request))
#19 [internal function]: Illuminate\Routing\Pipeline->Illuminate\Routing\{closure}(Object(Illuminate\Http\Request))
#20 D:\talentsaga\vendor\laravel\framework\src\Illuminate\Pipeline\Pipeline.php(103): call_user_func(Object(Closure), Object(Illuminate\Http\Request))
#21 D:\talentsaga\vendor\laravel\framework\src\Illuminate\Routing\ControllerDispatcher.php(96): Illuminate\Pipeline\Pipeline->then(Object(Closure))
#22 D:\talentsaga\vendor\laravel\framework\src\Illuminate\Routing\ControllerDispatcher.php(54): Illuminate\Routing\ControllerDispatcher->callWithinStack(Object(App\Http\Controllers\Frontend\HomeController), Object(Illuminate\Routing\Route), Object(Illuminate\Http\Request), 'getIndex')
#23 D:\talentsaga\vendor\laravel\framework\src\Illuminate\Routing\Route.php(174): Illuminate\Routing\ControllerDispatcher->dispatch(Object(Illuminate\Routing\Route), Object(Illuminate\Http\Request), 'App\\Http\\Contro...', 'getIndex')
#24 D:\talentsaga\vendor\laravel\framework\src\Illuminate\Routing\Route.php(140): Illuminate\Routing\Route->runController(Object(Illuminate\Http\Request))
#25 D:\talentsaga\vendor\laravel\framework\src\Illuminate\Routing\Router.php(724): Illuminate\Routing\Route->run(Object(Illuminate\Http\Request))
#26 [internal function]: Illuminate\Routing\Router->Illuminate\Routing\{closure}(Object(Illuminate\Http\Request))
#27 D:\talentsaga\vendor\laravel\framework\src\Illuminate\Routing\Pipeline.php(52): call_user_func(Object(Closure), Object(Illuminate\Http\Request))
#28 D:\talentsaga\app\Http\Middleware\Localization.php(52): Illuminate\Routing\Pipeline->Illuminate\Routing\{closure}(Object(Illuminate\Http\Request))
#29 [internal function]: App\Http\Middleware\Localization->handle(Object(Illuminate\Http\Request), Object(Closure))
#30 D:\talentsaga\vendor\laravel\framework\src\Illuminate\Pipeline\Pipeline.php(136): call_user_func_array(Array, Array)
#31 [internal function]: Illuminate\Pipeline\Pipeline->Illuminate\Pipeline\{closure}(Object(Illuminate\Http\Request))
#32 D:\talentsaga\vendor\laravel\framework\src\Illuminate\Routing\Pipeline.php(32): call_user_func(Object(Closure), Object(Illuminate\Http\Request))
#33 D:\talentsaga\vendor\laravel\framework\src\Illuminate\Foundation\Http\Middleware\VerifyCsrfToken.php(64): Illuminate\Routing\Pipeline->Illuminate\Routing\{closure}(Object(Illuminate\Http\Request))
#34 D:\talentsaga\app\Http\Middleware\VerifyCsrfToken.php(32): Illuminate\Foundation\Http\Middleware\VerifyCsrfToken->handle(Object(Illuminate\Http\Request), Object(Closure))
#35 [internal function]: App\Http\Middleware\VerifyCsrfToken->handle(Object(Illuminate\Http\Request), Object(Closure))
#36 D:\talentsaga\vendor\laravel\framework\src\Illuminate\Pipeline\Pipeline.php(136): call_user_func_array(Array, Array)
#37 [internal function]: Illuminate\Pipeline\Pipeline->Illuminate\Pipeline\{closure}(Object(Illuminate\Http\Request))
#38 D:\talentsaga\vendor\laravel\framework\src\Illuminate\Routing\Pipeline.php(32): call_user_func(Object(Closure), Object(Illuminate\Http\Request))
#39 D:\talentsaga\vendor\laravel\framework\src\Illuminate\View\Middleware\ShareErrorsFromSession.php(49): Illuminate\Routing\Pipeline->Illuminate\Routing\{closure}(Object(Illuminate\Http\Request))
#40 [internal function]: Illuminate\View\Middleware\ShareErrorsFromSession->handle(Object(Illuminate\Http\Request), Object(Closure))
#41 D:\talentsaga\vendor\laravel\framework\src\Illuminate\Pipeline\Pipeline.php(136): call_user_func_array(Array, Array)
#42 [internal function]: Illuminate\Pipeline\Pipeline->Illuminate\Pipeline\{closure}(Object(Illuminate\Http\Request))
#43 D:\talentsaga\vendor\laravel\framework\src\Illuminate\Routing\Pipeline.php(32): call_user_func(Object(Closure), Object(Illuminate\Http\Request))
#44 D:\talentsaga\vendor\laravel\framework\src\Illuminate\Session\Middleware\StartSession.php(64): Illuminate\Routing\Pipeline->Illuminate\Routing\{closure}(Object(Illuminate\Http\Request))
#45 [internal function]: Illuminate\Session\Middleware\StartSession->handle(Object(Illuminate\Http\Request), Object(Closure))
#46 D:\talentsaga\vendor\laravel\framework\src\Illuminate\Pipeline\Pipeline.php(136): call_user_func_array(Array, Array)
#47 [internal function]: Illuminate\Pipeline\Pipeline->Illuminate\Pipeline\{closure}(Object(Illuminate\Http\Request))
#48 D:\talentsaga\vendor\laravel\framework\src\Illuminate\Routing\Pipeline.php(32): call_user_func(Object(Closure), Object(Illuminate\Http\Request))
#49 D:\talentsaga\vendor\laravel\framework\src\Illuminate\Cookie\Middleware\AddQueuedCookiesToResponse.php(37): Illuminate\Routing\Pipeline->Illuminate\Routing\{closure}(Object(Illuminate\Http\Request))
#50 [internal function]: Illuminate\Cookie\Middleware\AddQueuedCookiesToResponse->handle(Object(Illuminate\Http\Request), Object(Closure))
#51 D:\talentsaga\vendor\laravel\framework\src\Illuminate\Pipeline\Pipeline.php(136): call_user_func_array(Array, Array)
#52 [internal function]: Illuminate\Pipeline\Pipeline->Illuminate\Pipeline\{closure}(Object(Illuminate\Http\Request))
#53 D:\talentsaga\vendor\laravel\framework\src\Illuminate\Routing\Pipeline.php(32): call_user_func(Object(Closure), Object(Illuminate\Http\Request))
#54 D:\talentsaga\vendor\laravel\framework\src\Illuminate\Cookie\Middleware\EncryptCookies.php(59): Illuminate\Routing\Pipeline->Illuminate\Routing\{closure}(Object(Illuminate\Http\Request))
#55 [internal function]: Illuminate\Cookie\Middleware\EncryptCookies->handle(Object(Illuminate\Http\Request), Object(Closure))
#56 D:\talentsaga\vendor\laravel\framework\src\Illuminate\Pipeline\Pipeline.php(136): call_user_func_array(Array, Array)
#57 [internal function]: Illuminate\Pipeline\Pipeline->Illuminate\Pipeline\{closure}(Object(Illuminate\Http\Request))
#58 D:\talentsaga\vendor\laravel\framework\src\Illuminate\Routing\Pipeline.php(32): call_user_func(Object(Closure), Object(Illuminate\Http\Request))
#59 [internal function]: Illuminate\Routing\Pipeline->Illuminate\Routing\{closure}(Object(Illuminate\Http\Request))
#60 D:\talentsaga\vendor\laravel\framework\src\Illuminate\Pipeline\Pipeline.php(103): call_user_func(Object(Closure), Object(Illuminate\Http\Request))
#61 D:\talentsaga\vendor\laravel\framework\src\Illuminate\Routing\Router.php(726): Illuminate\Pipeline\Pipeline->then(Object(Closure))
#62 D:\talentsaga\vendor\laravel\framework\src\Illuminate\Routing\Router.php(699): Illuminate\Routing\Router->runRouteWithinStack(Object(Illuminate\Routing\Route), Object(Illuminate\Http\Request))
#63 D:\talentsaga\vendor\laravel\framework\src\Illuminate\Routing\Router.php(675): Illuminate\Routing\Router->dispatchToRoute(Object(Illuminate\Http\Request))
#64 D:\talentsaga\vendor\laravel\framework\src\Illuminate\Foundation\Http\Kernel.php(246): Illuminate\Routing\Router->dispatch(Object(Illuminate\Http\Request))
#65 [internal function]: Illuminate\Foundation\Http\Kernel->Illuminate\Foundation\Http\{closure}(Object(Illuminate\Http\Request))
#66 D:\talentsaga\vendor\laravel\framework\src\Illuminate\Routing\Pipeline.php(52): call_user_func(Object(Closure), Object(Illuminate\Http\Request))
#67 D:\talentsaga\vendor\barryvdh\laravel-debugbar\src\Middleware\Debugbar.php(51): Illuminate\Routing\Pipeline->Illuminate\Routing\{closure}(Object(Illuminate\Http\Request))
#68 [internal function]: Barryvdh\Debugbar\Middleware\Debugbar->handle(Object(Illuminate\Http\Request), Object(Closure))
#69 D:\talentsaga\vendor\laravel\framework\src\Illuminate\Pipeline\Pipeline.php(136): call_user_func_array(Array, Array)
#70 [internal function]: Illuminate\Pipeline\Pipeline->Illuminate\Pipeline\{closure}(Object(Illuminate\Http\Request))
#71 D:\talentsaga\vendor\laravel\framework\src\Illuminate\Routing\Pipeline.php(32): call_user_func(Object(Closure), Object(Illuminate\Http\Request))
#72 D:\talentsaga\vendor\laravel\framework\src\Illuminate\Foundation\Http\Middleware\CheckForMaintenanceMode.php(44): Illuminate\Routing\Pipeline->Illuminate\Routing\{closure}(Object(Illuminate\Http\Request))
#73 [internal function]: Illuminate\Foundation\Http\Middleware\CheckForMaintenanceMode->handle(Object(Illuminate\Http\Request), Object(Closure))
#74 D:\talentsaga\vendor\laravel\framework\src\Illuminate\Pipeline\Pipeline.php(136): call_user_func_array(Array, Array)
#75 [internal function]: Illuminate\Pipeline\Pipeline->Illuminate\Pipeline\{closure}(Object(Illuminate\Http\Request))
#76 D:\talentsaga\vendor\laravel\framework\src\Illuminate\Routing\Pipeline.php(32): call_user_func(Object(Closure), Object(Illuminate\Http\Request))
#77 [internal function]: Illuminate\Routing\Pipeline->Illuminate\Routing\{closure}(Object(Illuminate\Http\Request))
#78 D:\talentsaga\vendor\laravel\framework\src\Illuminate\Pipeline\Pipeline.php(103): call_user_func(Object(Closure), Object(Illuminate\Http\Request))
#79 D:\talentsaga\vendor\laravel\framework\src\Illuminate\Foundation\Http\Kernel.php(132): Illuminate\Pipeline\Pipeline->then(Object(Closure))
#80 D:\talentsaga\vendor\laravel\framework\src\Illuminate\Foundation\Http\Kernel.php(99): Illuminate\Foundation\Http\Kernel->sendRequestThroughRouter(Object(Illuminate\Http\Request))
#81 D:\talentsaga\public\index.php(54): Illuminate\Foundation\Http\Kernel->handle(Object(Illuminate\Http\Request))
#82 {main}  
[2018-06-01 09:59:16] local.ERROR: exception 'ErrorException' with message 'file_put_contents(D:\talentsaga\storage\framework\views/e30f4707c2733c82417837275da98d494157f643.php): failed to open stream: Invalid argument' in D:\talentsaga\vendor\laravel\framework\src\Illuminate\Filesystem\Filesystem.php:111
Stack trace:
#0 [internal function]: Illuminate\Foundation\Bootstrap\HandleExceptions->handleError(2, 'file_put_conten...', 'D:\\talentsaga\\v...', 111, Array)
#1 D:\talentsaga\vendor\laravel\framework\src\Illuminate\Filesystem\Filesystem.php(111): file_put_contents('D:\\talentsaga\\s...', '<!doctype html>...', 0)
#2 D:\talentsaga\vendor\laravel\framework\src\Illuminate\View\Compilers\BladeCompiler.php(116): Illuminate\Filesystem\Filesystem->put('D:\\talentsaga\\s...', '<!doctype html>...')
#3 D:\talentsaga\vendor\laravel\framework\src\Illuminate\View\Engines\CompilerEngine.php(51): Illuminate\View\Compilers\BladeCompiler->compile('D:\\talentsaga\\r...')
#4 D:\talentsaga\vendor\laravel\framework\src\Illuminate\View\View.php(149): Illuminate\View\Engines\CompilerEngine->get('D:\\talentsaga\\r...', Array)
#5 D:\talentsaga\vendor\laravel\framework\src\Illuminate\View\View.php(120): Illuminate\View\View->getContents()
#6 D:\talentsaga\vendor\laravel\framework\src\Illuminate\View\View.php(85): Illuminate\View\View->renderContents()
#7 D:\talentsaga\vendor\laravel\framework\src\Illuminate\Http\Response.php(53): Illuminate\View\View->render()
#8 D:\talentsaga\vendor\symfony\http-foundation\Response.php(201): Illuminate\Http\Response->setContent(Object(Illuminate\View\View))
#9 D:\talentsaga\vendor\laravel\framework\src\Illuminate\Routing\ResponseFactory.php(57): Symfony\Component\HttpFoundation\Response->__construct(Object(Illuminate\View\View), 404, Array)
#10 D:\talentsaga\vendor\laravel\framework\src\Illuminate\Routing\ResponseFactory.php(71): Illuminate\Routing\ResponseFactory->make(Object(Illuminate\View\View), 404, Array)
#11 D:\talentsaga\vendor\laravel\framework\src\Illuminate\Foundation\Exceptions\Handler.php(156): Illuminate\Routing\ResponseFactory->view('errors.404', Array, 404, Array)
#12 D:\talentsaga\vendor\laravel\framework\src\Illuminate\Foundation\Exceptions\Handler.php(113): Illuminate\Foundation\Exceptions\Handler->renderHttpException(Object(Symfony\Component\HttpKernel\Exception\NotFoundHttpException))
#13 D:\talentsaga\app\Exceptions\Handler.php(52): Illuminate\Foundation\Exceptions\Handler->render(Object(Illuminate\Http\Request), Object(Symfony\Component\HttpKernel\Exception\NotFoundHttpException))
#14 D:\talentsaga\vendor\laravel\framework\src\Illuminate\Routing\Pipeline.php(80): App\Exceptions\Handler->render(Object(Illuminate\Http\Request), Object(Symfony\Component\HttpKernel\Exception\NotFoundHttpException))
#15 D:\talentsaga\vendor\laravel\framework\src\Illuminate\Routing\Pipeline.php(54): Illuminate\Routing\Pipeline->handleException(Object(Illuminate\Http\Request), Object(Symfony\Component\HttpKernel\Exception\NotFoundHttpException))
#16 D:\talentsaga\vendor\barryvdh\laravel-debugbar\src\Middleware\Debugbar.php(51): Illuminate\Routing\Pipeline->Illuminate\Routing\{closure}(Object(Illuminate\Http\Request))
#17 [internal function]: Barryvdh\Debugbar\Middleware\Debugbar->handle(Object(Illuminate\Http\Request), Object(Closure))
#18 D:\talentsaga\vendor\laravel\framework\src\Illuminate\Pipeline\Pipeline.php(136): call_user_func_array(Array, Array)
#19 [internal function]: Illuminate\Pipeline\Pipeline->Illuminate\Pipeline\{closure}(Object(Illuminate\Http\Request))
#20 D:\talentsaga\vendor\laravel\framework\src\Illuminate\Routing\Pipeline.php(32): call_user_func(Object(Closure), Object(Illuminate\Http\Request))
#21 D:\talentsaga\vendor\laravel\framework\src\Illuminate\Foundation\Http\Middleware\CheckForMaintenanceMode.php(44): Illuminate\Routing\Pipeline->Illuminate\Routing\{closure}(Object(Illuminate\Http\Request))
#22 [internal function]: Illuminate\Foundation\Http\Middleware\CheckForMaintenanceMode->handle(Object(Illuminate\Http\Request), Object(Closure))
#23 D:\talentsaga\vendor\laravel\framework\src\Illuminate\Pipeline\Pipeline.php(136): call_user_func_array(Array, Array)
#24 [internal function]: Illuminate\Pipeline\Pipeline->Illuminate\Pipeline\{closure}(Object(Illuminate\Http\Request))
#25 D:\talentsaga\vendor\laravel\framework\src\Illuminate\Routing\Pipeline.php(32): call_user_func(Object(Closure), Object(Illuminate\Http\Request))
#26 [internal function]: Illuminate\Routing\Pipeline->Illuminate\Routing\{closure}(Object(Illuminate\Http\Request))
#27 D:\talentsaga\vendor\laravel\framework\src\Illuminate\Pipeline\Pipeline.php(103): call_user_func(Object(Closure), Object(Illuminate\Http\Request))
#28 D:\talentsaga\vendor\laravel\framework\src\Illuminate\Foundation\Http\Kernel.php(132): Illuminate\Pipeline\Pipeline->then(Object(Closure))
#29 D:\talentsaga\vendor\laravel\framework\src\Illuminate\Foundation\Http\Kernel.php(99): Illuminate\Foundation\Http\Kernel->sendRequestThroughRouter(Object(Illuminate\Http\Request))
#30 D:\talentsaga\public\index.php(54): Illuminate\Foundation\Http\Kernel->handle(Object(Illuminate\Http\Request))
#31 {main}  
[2018-06-01 09:59:18] production.ERROR: exception 'RuntimeException' with message 'No supported encrypter found. The cipher and / or key length are invalid.' in D:\talentsaga\vendor\laravel\framework\src\Illuminate\Encryption\EncryptionServiceProvider.php:45
Stack trace:
#0 D:\talentsaga\vendor\laravel\framework\src\Illuminate\Encryption\EncryptionServiceProvider.php(25): Illuminate\Encryption\EncryptionServiceProvider->getEncrypterForKeyAndCipher(NULL, 'AES-256-CBC')
#1 D:\talentsaga\vendor\laravel\framework\src\Illuminate\Container\Container.php(731): Illuminate\Encryption\EncryptionServiceProvider->Illuminate\Encryption\{closure}(Object(Illuminate\Foundation\Application), Array)
#2 D:\talentsaga\vendor\laravel\framework\src\Illuminate\Container\Container.php(629): Illuminate\Container\Container->build(Object(Closure), Array)
#3 D:\talentsaga\vendor\laravel\framework\src\Illuminate\Foundation\Application.php(697): Illuminate\Container\Container->make('encrypter', Array)
#4 D:\talentsaga\vendor\laravel\framework\src\Illuminate\Container\Container.php(849): Illuminate\Foundation\Application->make('Illuminate\\Cont...')
#5 D:\talentsaga\vendor\laravel\framework\src\Illuminate\Container\Container.php(804): Illuminate\Container\Container->resolveClass(Object(ReflectionParameter))
#6 D:\talentsaga\vendor\laravel\framework\src\Illuminate\Container\Container.php(775): Illuminate\Container\Container->getDependencies(Array, Array)
#7 D:\talentsaga\vendor\laravel\framework\src\Illuminate\Container\Container.php(629): Illuminate\Container\Container->build('App\\Http\\Middle...', Array)
#8 D:\talentsaga\vendor\laravel\framework\src\Illuminate\Foundation\Application.php(697): Illuminate\Container\Container->make('App\\Http\\Middle...', Array)
#9 D:\talentsaga\vendor\laravel\framework\src\Illuminate\Pipeline\Pipeline.php(126): Illuminate\Foundation\Application->make('App\\Http\\Middle...')
#10 [internal function]: Illuminate\Pipeline\Pipeline->Illuminate\Pipeline\{closure}(Object(Illuminate\Http\Request))
#11 D:\talentsaga\vendor\laravel\framework\src\Illuminate\Routing\Pipeline.php(32): call_user_func(Object(Closure), Object(Illuminate\Http\Request))
#12 [internal function]: Illuminate\Routing\Pipeline->Illuminate\Routing\{closure}(Object(Illuminate\Http\Request))
#13 D:\talentsaga\vendor\laravel\framework\src\Illuminate\Pipeline\Pipeline.php(103): call_user_func(Object(Closure), Object(Illuminate\Http\Request))
#14 D:\talentsaga\vendor\laravel\framework\src\Illuminate\Routing\Router.php(726): Illuminate\Pipeline\Pipeline->then(Object(Closure))
#15 D:\talentsaga\vendor\laravel\framework\src\Illuminate\Routing\Router.php(699): Illuminate\Routing\Router->runRouteWithinStack(Object(Illuminate\Routing\Route), Object(Illuminate\Http\Request))
#16 D:\talentsaga\vendor\laravel\framework\src\Illuminate\Routing\Router.php(675): Illuminate\Routing\Router->dispatchToRoute(Object(Illuminate\Http\Request))
#17 D:\talentsaga\vendor\laravel\framework\src\Illuminate\Foundation\Http\Kernel.php(246): Illuminate\Routing\Router->dispatch(Object(Illuminate\Http\Request))
#18 [internal function]: Illuminate\Foundation\Http\Kernel->Illuminate\Foundation\Http\{closure}(Object(Illuminate\Http\Request))
#19 D:\talentsaga\vendor\laravel\framework\src\Illuminate\Routing\Pipeline.php(52): call_user_func(Object(Closure), Object(Illuminate\Http\Request))
#20 D:\talentsaga\vendor\laravel\framework\src\Illuminate\Foundation\Http\Middleware\CheckForMaintenanceMode.php(44): Illuminate\Routing\Pipeline->Illuminate\Routing\{closure}(Object(Illuminate\Http\Request))
#21 [internal function]: Illuminate\Foundation\Http\Middleware\CheckForMaintenanceMode->handle(Object(Illuminate\Http\Request), Object(Closure))
#22 D:\talentsaga\vendor\laravel\framework\src\Illuminate\Pipeline\Pipeline.php(136): call_user_func_array(Array, Array)
#23 [internal function]: Illuminate\Pipeline\Pipeline->Illuminate\Pipeline\{closure}(Object(Illuminate\Http\Request))
#24 D:\talentsaga\vendor\laravel\framework\src\Illuminate\Routing\Pipeline.php(32): call_user_func(Object(Closure), Object(Illuminate\Http\Request))
#25 [internal function]: Illuminate\Routing\Pipeline->Illuminate\Routing\{closure}(Object(Illuminate\Http\Request))
#26 D:\talentsaga\vendor\laravel\framework\src\Illuminate\Pipeline\Pipeline.php(103): call_user_func(Object(Closure), Object(Illuminate\Http\Request))
#27 D:\talentsaga\vendor\laravel\framework\src\Illuminate\Foundation\Http\Kernel.php(132): Illuminate\Pipeline\Pipeline->then(Object(Closure))
#28 D:\talentsaga\vendor\laravel\framework\src\Illuminate\Foundation\Http\Kernel.php(99): Illuminate\Foundation\Http\Kernel->sendRequestThroughRouter(Object(Illuminate\Http\Request))
#29 D:\talentsaga\public\index.php(54): Illuminate\Foundation\Http\Kernel->handle(Object(Illuminate\Http\Request))
#30 {main}  
[2018-06-01 09:59:18] production.ERROR: exception 'RuntimeException' with message 'No supported encrypter found. The cipher and / or key length are invalid.' in D:\talentsaga\vendor\laravel\framework\src\Illuminate\Encryption\EncryptionServiceProvider.php:45
Stack trace:
#0 D:\talentsaga\vendor\laravel\framework\src\Illuminate\Encryption\EncryptionServiceProvider.php(25): Illuminate\Encryption\EncryptionServiceProvider->getEncrypterForKeyAndCipher(NULL, 'AES-256-CBC')
#1 D:\talentsaga\vendor\laravel\framework\src\Illuminate\Container\Container.php(731): Illuminate\Encryption\EncryptionServiceProvider->Illuminate\Encryption\{closure}(Object(Illuminate\Foundation\Application), Array)
#2 D:\talentsaga\vendor\laravel\framework\src\Illuminate\Container\Container.php(629): Illuminate\Container\Container->build(Object(Closure), Array)
#3 D:\talentsaga\vendor\laravel\framework\src\Illuminate\Foundation\Application.php(697): Illuminate\Container\Container->make('encrypter', Array)
#4 D:\talentsaga\vendor\laravel\framework\src\Illuminate\Container\Container.php(849): Illuminate\Foundation\Application->make('Illuminate\\Cont...')
#5 D:\talentsaga\vendor\laravel\framework\src\Illuminate\Container\Container.php(804): Illuminate\Container\Container->resolveClass(Object(ReflectionParameter))
#6 D:\talentsaga\vendor\laravel\framework\src\Illuminate\Container\Container.php(775): Illuminate\Container\Container->getDependencies(Array, Array)
#7 D:\talentsaga\vendor\laravel\framework\src\Illuminate\Container\Container.php(629): Illuminate\Container\Container->build('App\\Http\\Middle...', Array)
#8 D:\talentsaga\vendor\laravel\framework\src\Illuminate\Foundation\Application.php(697): Illuminate\Container\Container->make('App\\Http\\Middle...', Array)
#9 D:\talentsaga\vendor\laravel\framework\src\Illuminate\Foundation\Http\Kernel.php(152): Illuminate\Foundation\Application->make('App\\Http\\Middle...')
#10 D:\talentsaga\public\index.php(58): Illuminate\Foundation\Http\Kernel->terminate(Object(Illuminate\Http\Request), Object(Illuminate\Http\Response))
#11 {main}  
